litmus-portal/frontend/src/containers/layouts/InputField/index.tsx

```diff
@@ -13,16 +13,18 @@ interface InputFieldProps {
   helperText?: string;
   validationError: boolean;
   success?: boolean;
+  disabled?: boolean;
   value: string;
-  required: boolean;
+  required?: boolean;
   iconType?: string | undefined;
-  handleChange: (event: React.ChangeEvent<{ value: string }>) => void;
+  handleChange?: (event: React.ChangeEvent<{ value: string }>) => void;
```

Comment:
handleChange cannot be an optional parameter, input fields always change and receive an event and change it to HTMLInputElement